pygeoapi/linked_data.py

```diff
@@ -174,20 +177,31 @@ def geojson2geojsonld(config, data, dataset, identifier=None):
 
     :returns: string of rendered JSON (GeoJSON-LD)
     """
+
     context = config['resources'][dataset].get('context', [])
-    data['id'] = (
-        '{}/collections/{}/items/{}' if identifier
-        else '{}/collections/{}/items'
-    ).format(
-        *[config['server']['url'], dataset, identifier]
-    )
+
+    uri = data['properties'].get('uri') if 'properties' in data else None
```

Comment:
Will need to add documentation that `uri` is a magic property that has this effect; and probably allow configuration to override it.

Reply:
Actually, I wonder if we can better hook into the configuration `provider['id_field']`.

Reply:
I think it's a good idea to have this be a part of the config in some way, perhaps something akin to:
![image](https://user-images.githubusercontent.com/30809186/85751271-6d3b1080-b6d8-11ea-8b1f-fb60b780980c.png)
However, since you have the intention of doing a PR merging some of this, I probably want to finish sooner than later as the above warrants more discussion.  

Would documentation done as follows suffice or is there a preferred way to document this property:
![image](https://user-images.githubusercontent.com/30809186/85752367-5d6ffc00-b6d9-11ea-99ef-185dd65cbf69.png)

Reply:
With better English, ha.
![image](https://user-images.githubusercontent.com/30809186/85753384-349c3680-b6da-11ea-9574-1c97ad3539be.png)

Reply:
By documentation, I meant the user-facing configuration documentation in `/docs/source/configuration.rst`. As part of the merge I'm looking at elevating it be part of the configuration, so I can do this part too.

Reply:
Ok good to know, going forward I'll make sure to update the rst docs.